client/flutter/lib/pages/onboarding/onboarding_page.dart

```diff
@@ -23,21 +30,21 @@ class _OnboardingPageState extends State<OnboardingPage> {
   @override
   Widget build(BuildContext context) {
     return WillPopScope(
-      onWillPop: () {
-        if (this._pageController.hasClients && this._pageController.page > 0) {
-          this._pageController.previousPage(duration: _animationDuration, curve: _animationCurve);
+      onWillPop: () async {
```

Comment:
Why refactor this method? Does it have to do with the fix?

Reply:
Partially, it wasn't behaving properly before.